lib/tasks/fetch.rake

```diff
@@ -113,9 +113,9 @@ task fix_credits_count_cache: :environment do
 end
 
 task record_db_table_counts: :environment do
-  RecordDbTableCountsWorker.perform_async
+  Metrics::RecordDbTableCountsWorker.perform_async
```

Comment:
Fixes [this error](https://app.honeybadger.io/projects/66984/faults/59594123), whoops!